benchmarks/2018.json

```diff
@@ -5182,5 +5164,23 @@
       85.67,
       92.31
     ]
+  },
+  {
+    "measureId": "343",
+    "benchmarkYear": 2016,
```

Comment:
Shouldn't it be removed completely? Or do I not understand things right now

Reply:
yes you're right! reran build-benchmarks, fixed